docs/tutorials/04-deploy.md

```diff
@@ -26,71 +26,49 @@
-```
+* [`pre-merge.yaml`](.github/workflows/pre-merge.yaml) Builds the project, runs the linter and the tests on every Pull Request
+* [`deploy.yaml`](.github/workflows/deploy.yaml) Deploys the project to Firebase after every merge to `master`.
 
-To generate the `secure` value do the following steps:
+The `pre-merge` workflow is already configured and will work out of the box, once you fork the hoverboard repo.
+To run the `deploy` on your instance instead, you need to do a small setup:
 
-1. Log in to Firebase console
+#### Deploying to Firebase wiht Github Actions
 
-   ```console
-     npx firebase login:ci --interactive
-   ```
+First, make sure that you're able to deploy locally correctly with the command:
 
-   You will get your token:
-
-   ```console
-       ✔  Success! Use this token to log in on a CI server:
-
-       1/9YmsNEh87G3cRyt_FXQbsYI_uV4FUMmUBXkbl_CHANGED
-   ```
-
-1. Install travis tool to encrypt the token
-
-   ```console
-     gem install travis
-   ```
-
-1. Log in to your account
-
-   ```console
-     travis login --auto
-   ```
-
-1. Encrypt your token
-
-   ```console
-     travis encrypt "1/9YmsNEh87G3cRyt_FXQbsYI_uV4FUMmUBXkbl_CHANGED"
-   ```
-
-   Approximate output:
+    ```console
+      npm run deploy
+    ```
 
-   ```console
-     secure: "cioDQ571EZpnuGiDn7ofvEghNFP82vz7N+SqIL5ZjOK0CBgaWO3OoePoh1eO1dvIsdLDr7yNs5kBIx8NIuOqUA9YLyIIasC7ah1QLtiK5zRVaCcgwt4aBqRLKJVbXPl08MIyk9GFYl2+J+oLOzoEOnVUuCpUcGYWdmDRTKis5KP6naK1msRmTu5ymQn55cyxpmSZS2F+iEsAgV7d0/h+HGgPPd77M26j8wV9JEFJp3iMhudaCkWdoBf9z9WP0cpPzTHgSHEU/Mski4oMfU1BqCFRiaKfcw/uLzMcTpjcf+YG2dc3qTMcuBNKNvhANnaYrxePtuW1VWb+xl19qVQWrsGpQgyWIbp+icSXF3KGR1wfNrC9zNQWKm112BckYn6id8w4M3JeRdWRaCwWitG9C5CWQ3ZepPpgBu2SYSfZQg5heIbVSYOgbXUfeR8ByJqyAGCrYrB3lyyR49cr+GAnILbOgxE7FRYuHmagLD+xa8cHUFcZUu6CxgrhOFa+28Lvrtvod1WqbIioZfhWRcdIZNdJxR4gxXaGycp5n0qjJ0o1VDFAUcy93ImYyVZFY+OmqfVLFQChAD9NnPT1a0v3gHYR3IMd5aXXtbOo9e6cAjuXU/NQCry10Y0bNiMKkHbvnj3aGfAWlA34CRj3iOK2Nz1udDwBMdUKsgt1xiVh3h8="
-   ```
+If the deploy is successful, you can then configure Github Actions to do it for you.
+You need to generate a login token for the CI with the following command:
 
-1. Replace generated encrypted token with existing one
+    ```console
+      npx firebase login:ci
+    ```
 
-1. **Tip:** deploy different builds depending on the branch:
-   ```yaml
-   script:
-     - echo "Building..."
-     - if [ "$TRAVIS_BRANCH" == "master" ]; then npm run build; else BUILD_ENV=development npm run build; fi
-   ```
-1. Push to a repository
+Once you obtained the token, you need to store it as a **secret** called `FIREBASE_TOKEN` in the settings of your repository.
```

Comment:
It would be neat if we could use `GCP_SA_KEY` instead and have less privileged secrets. It looks though like creating the keys is much more involved than Firebase tokens.

Reply:
I think that the `FIREBASE_TOKEN` approach is easier to setup (so I'd stick to that for the `docs` file). If you wish, I can change our workflow file to use `GCP_SA_KEY` and maybe leave a comment in the `docs` file mentioning that setup with a Service Account is also possible and link to:
https://github.com/marketplace/actions/github-action-for-firebase

Reply:
I'd rather stick to a single approach. So `FIREBASE_TOKEN` should be it.